tests/wpt/verifydnslength_tests.json

```diff
@@ -20,31 +20,6 @@
     "input": "http://br.exampleexampleexampleexampleexampleexampleexampleexampleexampleexample.lorenlorenlorenlorenlorenlorenlorenlorenlorenlorenlorenlorenlore",
     "failure": true
   },
-  {
```

Comment:
I think we should keep these tests, but change the testing code so that we call  `url.is_valid_domain()` on the parsed url.